src/pages/grants/grantsPage/sections/DistributionChart.tsx

```diff
@@ -19,26 +20,27 @@ export const DistributionChart = ({ applicants, isCompetitionVote }: Props) => {
     return prev + (curr?.funding.communityFunding || 0)
   }, 0)
 
-  const percentages: Array<GrantApplicant & { percentage: number; numberOfContributors: number }> = applicants.map(
-    (applicant) => ({
-      ...applicant,
-      percentage: ((applicant.funding?.communityFunding || 0) * 100) / (total || 1),
-      numberOfContributors: applicant.contributorsCount,
-    }),
-  )
+  const percentages: Array<
+    GrantApplicant & { percentage: number; numberOfContributors: number; communityFundingAmount: number }
+  > = applicants.map((applicant) => ({
+    ...applicant,
+    percentage: ((applicant.funding?.communityFunding || 0) * 100) / (total || 1),
+    numberOfContributors: applicant.contributorsCount,
+    communityFundingAmount: applicant.funding.communityFunding,
+  }))
 
   const maxPercentage = Math.max(...percentages.map((p) => p.percentage))
 
   return (
     <CardLayout noMobileBorder p={{ base: 0, lg: 5 }} w="full">
-      <H2>{t('Grant distribution status')}</H2>
```

Comment:
This only applies to grants of type `COMPETITION_VOTE`, the `COMMUNITY_VOTE` grants would still display grant distribution status.

Reply:
Yes I thought of that, but I thought Leaderboard still makes sense for both. Will update it though.